packages/connectivity/src/scp-cf/environment-accessor/environment-accessor-types.ts

```diff
@@ -19,6 +21,10 @@ export interface Service {
    * The service credentials.
    */
   credentials: ServiceCredentials;
+  /**
+   * IAS Resource to request for App-to-App communication.
+   */
+  iasResource?: IasResource | IasResource[];
```

Comment:
[q] This a representation of a service, VCAP_SERVICES in BTP, and the `iasResource` property doesn't fit in this interface.